server/cherry_pick.go

```diff
@@ -204,20 +211,22 @@ func (s *Server) doCherryPick(ctx context.Context, version string, milestoneNumb
 	cmd.Env = append(
 		os.Environ(),
 		os.Getenv("PATH"),
-		fmt.Sprintf("DRY_RUN=%t", false),
```

Comment:
@isacikgoz - Adding this was incorrect. :P The `DRY_RUN` env var just checks whether it's empty or not. Setting it to false would actually enable dry run. 

It's good that we didn't deploy yet. :D

Reply:
We need to go over the script and simplify it 😅